$OpenBSD: patch-meson_build,v 1.2 2021/04/24 09:30:50 ajacoutot Exp $

No ucontext.h on OpenBSD.

We only have zlib version 1.2.3 in base.

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -44,7 +44,7 @@ cc = meson.get_compiler('c')
 coroutine = get_option('with-coroutine')
 
 if coroutine == 'auto'
-  if host_machine.system() == 'windows'
+  if host_machine.system() == 'windows' or host_machine.system() == 'openbsd'
     coroutine = 'gthread'
   else
     coroutine = 'ucontext'
@@ -63,7 +63,7 @@ glib_min_version_symbol='GLIB_VERSION_2_56'
 gdk_pixbuf_min_version = '2.36.0'
 gcrypt_min_version = '1.8.0'
 gnutls_min_version = '3.6.0'
-zlib_min_version = '1.2.11'
+zlib_min_version = '1.2.3'
 libsasl_min_version = '2.1.27'
 
 # gtk-vnc
